src/actions/assetsActions.js

```diff
@@ -600,30 +600,23 @@ export const fetchAllAccountsTotalBalancesAction = () => {
   };
 };
 
-export const fetchAssetsBalancesAction = () => {
+export const fetchAssetsBalancesAction = (isRefresh?: boolean) => {
```

Comment:
Is there a better name that we can use instead of `isRefresh`? Something a little bit more clear - is it currently refreshing, or is it checking if the data has been refreshed?

Reply:
New name is isRefreshingPart. changes done.